Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -1037,6 +1559,92 @@ class CredentialsManagerSpec: QuickSpec {
 
             }
 
+            context("api credentials") {
```

Comment:
These are the tests for the Combine wrapper. Combine is Apple's framework for functional reactive programming.